### PR TITLE
neighborhood characteristics where affordable family sized units are …

### PR DESCRIPTION
**Pull request by maggiesu0725**
…located

### CODE REVIEW

**Comment by maggiesu0725** [comment]
@lhendey @rpitingolo  code for Table 1B ready for review

File: Prog/Lgunit- ACS Table 1B.sas [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+data calculate_pct;
+     set ACS.acs_2012_16_dc_sum_tr_tr10;
+	 keep geo2010 numrenterhsgunits_&_years. numownocchu3plusbd_&_years. numrentocchu3bd_&_years.
+          numrtohu3b500to749_&_years. numrtohu3b750to999_&_years.  numrtohu3bunder500_&_years. numrtohu3b1500plus_&_years.
+          pct3baffordable1000 pct3baffordable1500 aff1000median aff1000threequarter aff1500median aff1500threequarter;
+
+          pct3baffordable1500= (numrentocchu3bd_&_years.-numrtohu3b1500plus_&_years.)/numrentocchu3bd_&_years.;
+          pct3baffordable1000= sum(numrtohu3b500to749_&_years.,numrtohu3b750to999_&_years., numrtohu3bunder500_&_years.)/numrentocchu3bd_&_years.;
+     if   pct3baffordable1000 >=0.5 then aff1000median=1;
```
Comment (lhendey) [inline Prog/Lgunit- ACS Table 1B.sas]: Yipeng - these need to be based on the median and 75th percentile  for the indicator across tracts not 50% and 75%. You can run a proc univariate to get those values.

File: Prog/Lgunit- ACS Table 1B.sas [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+          pop25andoverwcollege_&_years. pop25andoverwouths_&_years. popunemployed_&_years. poppoorpersons_&_years. totpop_&_years. 
+	      famincomelt75k_&_years. numfamilies_&_years. nonfamilyhhtot_&_years. medfamincm_&_years. numhshlds_&_years.
+		  pop25andoveryears&_years. popincivlaborforce_&_years.	numrenteroccupiedhu_&_years.	
+          NumRtOHU1u_&_years. NumRtOHU2to4u_&_years. NumRtOHU5to9u_&_years. NumRtOHU10to19u_&_years. NumRtOHU20plusu_&_years.
+          pctnonwht pcthispan pctnonhisblk pctcollege pctwouths pctunemployed pctpoverty pctfambelow75000 pctnonfam
+		  rentersinglefam renter2to4 renter5to9 renter10to19 renter20plus 
+
+;  
+
+pctnonwht= (popwithrace_&_years. -popalonew_&_years.)/popwithrace_&_years.*100;
```
Comment (lhendey) [inline Prog/Lgunit- ACS Table 1B.sas]: there is a var for nonhispanic white use that instead of white alone

File: Prog/Lgunit- ACS Table 1B.sas [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+pcthispan= (popaloneh_&_years.)/popwithrace_&_years.*100;
+pctnonhisblk= (popblacknonhispbridge_&_years.)/popwithrace_&_years.*100;
+pctcollege= (pop25andoverwcollege_&_years.)/pop25andoveryears_&_years.*100;
+pctwouths= (pop25andoverwouths_&_years.)/pop25andoveryears_&_years.*100;
+pctunemployed= (popunemployed_&_years.)/popincivlaborforce_&_years.*100;
+pctpoverty= (poppoorpersons_&_years.)/totpop_&_years. *100;
```
Comment (lhendey) [inline Prog/Lgunit- ACS Table 1B.sas]: you need pop with poverty defined as the denominator

File: Prog/Lgunit- ACS Table 1B.sas [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+     set Police.Crimes_sum_tr10;
+	 keep geo2010 crimes_pt1_property_2017 crimes_pt1_violent_2017 crime_rate_pop_2017 pctpropertycrime pctviolentcrime ;
+     pctpropertycrime= crimes_pt1_property_2017/crime_rate_pop_2017*100;
+     pctviolentcrime=crimes_pt1_violent_2017/crime_rate_pop_2017*100;
+	 ;
+run;
+
+data prenatal;
+     set vital.Births_2016;
+     keep births_prenat_adeq births_total pctprenatal geo2010;
```
Comment (lhendey) [inline Prog/Lgunit- ACS Table 1B.sas]: double check there is not a denominator that is births with prenatal care defined.

File: Prog/Lgunit- ACS Table 1B.sas [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+     set realprop.sales_res_clean ;
+     keep geo2010 saleprice saledate saleyear ;
+	 saleyear=year(saledate)
+	 ;
+run;
+proc sort data= ressale;
+by geo2010;
+run;
+
+proc means median data = ressale (where=(saleyear=2017));
```
Comment (lhendey) [inline Prog/Lgunit- ACS Table 1B.sas]: i don't think median is appearing on the output file.

Comment (lhendey) [reply Prog/Lgunit- ACS Table 1B.sas]: also the output file has several rows per tract and isn't merging correctly later.

**Comment by maggiesu0725** [comment]
@lhendey  all comments have been addressed. Code ready for review.

**Comment by lhendey** [comment]
@maggiesu0725 - thanks. can you fill in 1C?  also did you see the email I sent last night with the data from Ruthy? We can leave this branch open if you want to add that to this program.

**Comment by maggiesu0725** [comment]
@lhendey Yes I will fill in 1C now and update the program to add the HUD data, maybe tomorrow

**Comment by lhendey** [comment]
@maggiesu0725 that would be great, thanks.

**Comment by lhendey** [comment]
@maggiesu0725 - did you get a chance to add the HUD data?

**Comment by maggiesu0725** [comment]
@lhendey working on it now.

**Comment by maggiesu0725** [comment]
@lhendey added HUD data and ready for review

**Comment by lhendey** [comment]
@maggiesu0725 please go head and merge.

**Pull request merged**